feature:关联 namespace界面，可以一次选择多个namespace (#4437)
apolloconfig/apollo: CHANGES.md

* [Upgrade mysql-connector-java version to fix possible transaction rollback failure issue](https://github.com/apolloconfig/apollo/pull/4425)
* [Remove database migration tool Flyway](https://github.com/apolloconfig/apollo/pull/4361)
* [Optimize Spring-Security Firewall Deny Request Response 400](https://github.com/apolloconfig/apollo/pull/4428)

> * [On the Namespace association screen, you can select multiple namespaces at a time](https://github.com/apolloconfig/apollo/pull/4437)

Reviewer: ```suggestion
* [Allow users to associate multiple public namespaces at a time](https://github.com/apolloconfig/apollo/pull/4437)
```
